lesson1/src/stacks_queues/stack.js

```diff
@@ -9,20 +9,34 @@ class Stack {
    */
   push(value) {
     // todo: do stuff
+    this.stack[this.top] = value
+    this.top++
   }
-
+ 
   /*
    Pop should return the value and remove it from the queue
    */
   pop() {
     // todo: do stuff
+    if (this.top <= 0) {
+      return "Empty Stack"
+    }
+    let newStack = []
```

Comment:
ill let you use `splice` to delete the element and shift the others ! (but its nice that you now notice splice is O(n))